lib/ReactViews/StandardUserInterface/StandardUserInterface.jsx

```diff
@@ -248,7 +248,7 @@ const StandardUserInterface = observer(
         !this.shouldUseMobileInterface();
       const showStoryPanel =
         this.props.terria.configParameters.storyEnabled &&
-        this.props.terria.stories.length &&
+        this.props.terria.stories.length > 0 &&
```

Comment:
All conditionals used in react jsx should be true/false. In this case `showStoryPanel` is `0` when there are no story scenes. This resulted in a stray `0` to be printed in the dom.